docs/source/how_to_guides/configure_cloud_storage_cred.md

```diff
@@ -69,6 +72,29 @@ export MOSAICML_STREAMING_AWS_REQUESTER_PAYS='streaming-bucket,another-bucket'
 ```
 ````
 
+## S3 compatible object store
+For any S3 compatible object store such as [Cloudflare R2](https://www.cloudflare.com/products/r2/), [Coreweave](https://docs.coreweave.com/storage/object-storage), [Backblaze b2](https://www.backblaze.com/b2/cloud-storage.html), etc., setup your credentials as mentioned in the above `Amazon S3` section. The only difference is you must set your object store endpoint url. To do this, you need to set the ``S3_ENDPOINT_URL`` environment variable.
+
+Below is one of such example on setting a R2 `endpoint url` in your run environment.
+
+```{note}
+Your endpoint url is `https://<accountid>.r2.cloudflarestorage.com`. The account ID can be retrieved through your [Cloudflare console](https://dash.cloudflare.com/).
+```
+
+````{tabs}
+```{code-tab} py
+import os
+os.environ['S3_ENDPOINT_URL'] = 'https://<accountid>.r2.cloudflarestorage.com'
+```
+
+```{code-tab} sh
+export S3_ENDPOINT_URL='https://<accountid>.r2.cloudflarestorage.com'
+```
+````
+
+The above step will add an environment variable `S3_ENDPOINT_URL` to your runs and the streaming dataset fetches those environment variables for authentication and stream data into your instance.
```

Comment:
and stream data -> to stream data?

Reply:
Fixed in https://github.com/mosaicml/streaming/pull/265/commits/de2d844124edd2be117f70455ca0edb8cddc5c23.